<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Javi\UBA\FIUBA\Algoritmos y Programación III\TP2\ TP2\ tp-algoritmos-tres\TP2\bin\Debug\TP2.dll" total="3" errors="2" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2012-05-28" time="17:03:10">
  <environment nunit-version="2.6.0.12051" clr-version="2.0.50727.3082" os-version="Microsoft Windows NT 5.1.2600 Service Pack 3" platform="Win32NT" cwd="C:\Javi\UBA\FIUBA\Algoritmos y Programación III\TP2\ TP2\ tp-algoritmos-tres\TP2\bin\Debug" machine-name="HOME" user="Administrador" user-domain="HOME" />
  <culture-info current-culture="es-AR" current-uiculture="es-ES" />
  <test-suite type="Assembly" name="C:\Javi\UBA\FIUBA\Algoritmos y Programación III\TP2\ TP2\ tp-algoritmos-tres\TP2\bin\Debug\TP2.dll" executed="True" result="Failure" success="False" time="0.188" asserts="0">
    <results>
      <test-suite type="Namespace" name="TP2" executed="True" result="Failure" success="False" time="0.172" asserts="0">
        <results>
          <test-suite type="TestFixture" name="CasilleroTest" executed="True" result="Success" success="True" time="0.078" asserts="0">
            <results>
              <test-case name="TP2.CasilleroTest.PruebaLaCreacionDeUnCasillero" executed="True" result="Success" success="True" time="0.031" asserts="2" />
            </results>
          </test-suite>
          <test-suite type="Namespace" name="Tests" executed="True" result="Failure" success="False" time="0.094" asserts="0">
            <results>
              <test-suite type="TestFixture" name="TableroTest" executed="True" result="Failure" success="False" time="0.094" asserts="0">
                <results>
                  <test-case name="TP2.Tests.TableroTest.PruebaCrearUnTableroConDimension1" executed="True" result="Error" success="False" time="0.016" asserts="0">
                    <failure>
                      <message><![CDATA[System.IndexOutOfRangeException : Index was outside the bounds of the array.]]></message>
                      <stack-trace><![CDATA[at TP2.Tablero..ctor(Int32 ancho, Int32 alto) in C:\Javi\UBA\FIUBA\Algoritmos y Programación III\TP2\ TP2\ tp-algoritmos-tres\TP2\src\Tablero.cs:line 23
at TP2.Tests.TableroTest.PruebaCrearUnTableroConDimension1() in C:\Javi\UBA\FIUBA\Algoritmos y Programación III\TP2\ TP2\ tp-algoritmos-tres\TP2\Tests\TableroTest.cs:line 15
]]></stack-trace>
                    </failure>
                  </test-case>
                  <test-case name="TP2.Tests.TableroTest.PruebaQueDevuelvaElCasilleroDeUnaPosicionDada" executed="True" result="Error" success="False" time="0.000" asserts="0">
                    <failure>
                      <message><![CDATA[System.IndexOutOfRangeException : Index was outside the bounds of the array.]]></message>
                      <stack-trace><![CDATA[at TP2.Tablero..ctor(Int32 ancho, Int32 alto) in C:\Javi\UBA\FIUBA\Algoritmos y Programación III\TP2\ TP2\ tp-algoritmos-tres\TP2\src\Tablero.cs:line 23
at TP2.Tests.TableroTest.PruebaQueDevuelvaElCasilleroDeUnaPosicionDada() in C:\Javi\UBA\FIUBA\Algoritmos y Programación III\TP2\ TP2\ tp-algoritmos-tres\TP2\Tests\TableroTest.cs:line 23
]]></stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>